### PR TITLE
added context (audience, purpose) to training material subsection

### PR DESCRIPTION
**Pull request by hvwaldow**
I think a short remark to make the distinction between teaching material **used by RSEs to teach** and resources useful for RSEs to **learn** is in order, even though there is obviously a lot of overlap.

### CODE REVIEW

File: paper_teaching-learning-RSE.md [hunk 1]
```diff
@@ -214,6 +220,11 @@ Reference works are also available for self-study [@Fogel2005], [@Irving2021].
 
 The National Competence Center Sweden (ENCCS) [https://enccs.se/] provides instructor training material [@ENCCSInstructorTraining], [@ENCCS2022] developed from Carpentries and CodeRefinery material, as well as lessons for HPC-oriented RSEs [@ENCCSLessons].
 
+From the MIT Computer Science & Artificial Intelligence Laboratory comes an
+unofficial course [The Missing Semester of Your CS
+Education](https://missing.csail.mit.edu/) which covers a lot of basic computer
+skills typically taught by RSEs.
```
Comment (jcohen02) [inline paper_teaching-learning-RSE.md:226]: I really like the addition of the link to this material. Maybe also add a note that these skills are important for anyone wanting to become an RSE? e.g.
```suggestion
skills typically taught by RSEs. The skills covered here also provide an important set of core capabilities for anyone looking to become an RSE.
```

**Review by jcohen02 — CHANGES_REQUESTED** [review]
Great addition @hvwaldow - spotted a typo and added a couple of suggestions, hence requesting changes, but otherwise this looks good.

File: paper_teaching-learning-RSE.md (hunk 1 above)
Comment (hvwaldow) [reply paper_teaching-learning-RSE.md]: That this part ended here was some sort of runtime-condition and my bad branch management. It's a seperate PR: #65

**Comment by hvwaldow** [comment]
@CaptainSifff I think this has been all merged now (contd. in #65) an therefore close this PR.

**Pull request closed without merging**